src/cpp/process_image.cpp

```diff
@@ -47,10 +48,15 @@ std::string ProcessImage::get_processed_image()
         cachePath = cachePath.substr(0, cachePath.find_last_of("/"));
         cachePath += "/Library/Caches/";
     }
+    else if (getOsName() == "Linux")
+    {
+        // Set cache path
+        cachePath = "/data/user/0/com.blossomscam/cache/";
+    }
```

Comment:
Looks like Android uses a Linux OS, got it!

Reply:
That is what it was giving me when running in the simulator, I never tried it on an actual device